client/tests/integration/queries/role.rs

```diff
@@ -38,8 +36,8 @@ fn find_roles() -> Result<()> {
 
     // Checking results
     let found_role_ids = test_client
-        .request(client::role::all())?
-        .collect::<QueryResult<Vec<_>>>()?
+        .query(client::role::all())
+        .execute_all()?
```

Comment:
No, not really.

Looking at prior art, diesel has `load` which collects into a `Vec` and `load_iter` which gives you an iterator. 

I like the `_all` name more (as it specifies _what_ it does instead of specifying what type it returns), but I am not very averse to the `into_vec` name